crates/store/re_chunk_store/src/gc.rs

```diff
@@ -513,6 +513,7 @@ impl ChunkStore {
         // and slow. Rather we need to surgically remove the superfluous chunks.
 
         let mut chunk_ids_removed = HashSet::default();
+        let mut row_ids_removed = HashSet::default();
```

Comment:
name is confusing to me, expected this to be _all_ row ids.
```suggestion
        let mut min_row_ids_removed  = HashSet::default();
```